db.json

```diff
@@ -464,6 +464,44 @@
         }
       ]
     },
+    {
+      "id": "webidl",
+      "name": "Web IDL",
+      "scope": "A syntax for specifications to use to define web APIs and the translation of that syntax into web browser programming languages such as JavaScript.",
+      "editors": [
+        {
+          "name": "Edgar Chen",
+          "email": "echen@mozilla.com"
+        },
+        {
+          "name": "Timothy Gu",
+          "email": "timothygu99@gmail.com"
+        }
+      ],
+      "standards": [
+        {
+          "name": "Web IDL",
+          "href": "https://webidl.spec.whatwg.org/",
+          "description": "The Web IDL Standard defines an interface definition language, Web IDL, that can be used to describe interfaces that are intended to be implemented in web browsers.",
+          "authors": [
+            {
+              "name": "Edgar Chen",
+              "email": "echen@mozilla.com"
+            },
+            {
+              "name": "Timothy Gu",
+              "email": "timothygu99@gmail.com"
+            }
+          ],
```

Comment:
Should we also add heycam to the list of authors here?

Reply:
Last I checked @heycam was not interested in maintaining it. Once we get this merged we'll work with him on transferring the repository and putting redirects in place.